charts/warpstream-agent/values.yaml

```diff
@@ -142,6 +142,14 @@ config:
   # Agent Groups configuration https://docs.warpstream.com/warpstream/byoc/advanced-agent-deployment-options/agent-groups
   # Leave empty for Default group
   agentGroup: ""
+  # Low Latency Cluster Settings
```

Comment:
maybe also mention that if these are set, bucketURL cant be set above?

Reply:
done